openid-sharedsignals-framework-1_0.md

```diff
@@ -798,10 +798,24 @@ events_delivered
 
 delivery
 
-> **Receiver-Supplied**, A JSON object containing a set of name/value pairs
-  specifying configuration parameters for the SET delivery method.  The actual
-  delivery method is identified by the special key "method" with the value being
-  a URI as defined in {{delivery-meta}}.
+> A JSON object containing a set of name/value pairs specifying configuration
+  parameters for the SET delivery method. The actual delivery method is
+  identified by the special key "method" with the value being a URI as defined
+  in {{delivery-meta}}. The value of the "delivery" field contains two
+  sub-fields: 
+  
+>   delivery_method
```

Comment:
`delivery_` is redundant as it's part of the `delivery` object. Also changing the attribute name would break the backwards compatibility.

I suggest we maintain `method` as an inner attribute name

Reply:
Without understanding the goal of this change, I agree with @appsdesh.

Reply:
The current spec uses "delivery_method" and not "method", although "method" is mentioned in the profile section. The intent of this PR is to clarify that in case of POLL, the "url" property should be supplied by the Transmitter. I am therefore not changing the name from "delivery_method" to "method" in this PR. We can have a separate PR to cover that.

Reply:
Actually, it looks like the examples give the wrong name "delivery_method", instead of the correct one "method". I will modify the PR to use "method". Stay tuned.